### DISCUSSION TITLE
Python code block appear different style within the same markdown page

### DISCUSSION THREAD

**Discussion by TonyFPY** [CaiJimmy/hugo-theme-stack#598] [General]
Is it because the first code block is not identified as python code?

![makrdown](https://user-images.githubusercontent.com/34860220/166154899-81371fcc-4c20-4ffc-90cf-4fed68102c40.png)![web page](https://user-images.githubusercontent.com/34860220/166154908-bee8e088-39aa-4a12-8646-4da19f5e8bb4.png)

**Comment by CaiJimmy** [comment]
> Is it because the first code block is not identified as python code?

Seems like that. It's recommended to indicate the language used.
